Client/multiplayer_sa/CMultiplayerSA_FrameRateFixes.cpp

```diff
@@ -586,6 +586,10 @@ static void __declspec(naked) HOOK_CWeapon_Update()
 {
     _asm
     {
+        // Temp fix for camera
```

Comment:
It tells that this fix is temporary. Can we anticipate subsequent improvements?

Reply:
The proper fix will come when the function is rewritten using our own implementation, which probably won’t happen anytime soon, so this is a temporary solution